bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -659,6 +659,7 @@ void BCStateTran::onTimerImp() {
   FetchingState fs = getFetchingState();
   if (fs == FetchingState::GettingCheckpointSummaries) {
     if ((currTime - lastTimeSentAskForCheckpointSummariesMsg) > config_.checkpointSummariesRetransmissionTimeoutMs) {
+      LOG_DEBUG(getLogger(), "Retransmitting AskForCheckpointSummaries");
```

Comment:
The main reason I would add a log here and not inside sendAskForCheckpointSummariesMsg (there is one there already, but just printing vars), is to print values of retransmissionNumberOfAskForCheckpointSummariesMsg, kResetCount_AskForCheckpointSummaries.
It's easy to find a send without this by searching `sendAskForCheckpointSummariesMsg`